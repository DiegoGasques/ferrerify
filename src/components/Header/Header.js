import React from "react";
import Container from "../Container/Container";

export const Header = () => {
  return (
    <header className="Header">
      <Container>
        <h1 className="Header__title">Ferrerify</h1>
      </Container>
    </header>
  );
};

export default Header;
