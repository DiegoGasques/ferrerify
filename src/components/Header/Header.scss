.Header {
  background: $primary;
  color: $snow;
  padding: $m-size;
  margin-bottom: $s-size;
  -webkit-box-shadow: $main-shadow;
  box-shadow: $main-shadow;
}

.Header h1 {
  font-weight: 500;
  padding: 0 1.5rem;
  letter-spacing: 0.4rem;
}
