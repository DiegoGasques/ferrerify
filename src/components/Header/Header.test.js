import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import { Header } from "./Header";

describe("[COMPONENT] Header", () => {
  it("Tests for changes in the rendered component", () => {
    const wrapper = shallow(<Header />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
