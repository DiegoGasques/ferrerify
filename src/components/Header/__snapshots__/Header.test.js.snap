// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[COMPONENT] Header Tests for changes in the rendered component 1`] = `
<header
  className="Header"
>
  <Container>
    <h1
      className="Header__title"
    >
      Ferrerify
    </h1>
  </Container>
</header>
`;
